Topics/Professionalism.md

```diff
@@ -67,6 +67,29 @@ Professionalism is essential in all aspects of software engineering, and maintai
+2. **Avoid being overly defensive and overwhelmed by taking a step back if the criticism seems too harsh.** You can step back in various ways, but we recommend being honest and communicating that you need a few minutes to compose yourself. This pause will also give the critic time to adjust their tone and thoughts so that the conversation can be more productive and beneficial to everyone involved.
+3. **Acknowledge valid points that the critic gives by stating what you agree with** (e.g. “I agree with you that this line of code here could be more readable”). Be very specific here so that there is no misunderstanding caused later on.
+4. **Listen attentively**, especially when they can see your face and body. What this means is to ensure that your body or, at least, your head is facing them, and ideally, you will give signs that show you are following along, such as nodding your head and so on. However, the more important thing is that you are paying attention to what the critic is saying and thinking about how you will respond using the first three tips to guide what you could think about.
+5. **Do your own additional research if necessary and fully understand the feedback before replying or making changes**, especially for code reviews. Following this advice shows others you are willing to take the initiative to improve instead of just being reactionary. However, if it seems like it will take too much time or if you are in a busy stretch, you may want to consult with the reviewer in question and ask for feedback directly, which may be more helpful, less prone to miscommunication, and faster. Especially for code, make sure to go back and check other code that you wrote to see if you can apply the same feedback and make improvements.
+6. **Summarize the feedback once you’ve gone through all the steps and believe you have a strong understanding.** In your summary, go over the problems that you’ve identified and the solutions and fixes you’ve agreed upon. If the person who provided you feedback is happy with your summary, then you can now go ahead and make the changes.
+7. **Bring in a mediator to sit in** if you feel that the critic does not want to have a productive conversation and is just personally attacking you. This mediator can be a co-worker, HR person, TA, etc.. The mediator should be someone who can calm the critic down and point out when either of you is making a comment designed to personally attack the other person. Make it clear when asking someone to be a mediator why you feel the need to bring them in.
```

Comment:
This advice on bringing in a mediator is valuable. But it might be helpful to add a bit more detail on how to choose an appropriate mediator and the best way to approach someone for this role. A brief note on what to do if a mediator is not available or if the mediation does not resolve the issue could also be beneficial for readers facing more complex situations.

Reply:
Thanks for the feedback! I've added a sentence on what to do when a mediator is not available, but I am wondering if you have any suggestions on your other point of how to select a mediator? I am not sure if there is general advice I can give on this latter point, so a suggestion would be appreciated.

Reply:
I'm glad to hear you've added content about what to do when a mediator is not available. For selecting a mediator, I believe the key is to choose someone neutral and unbiased, with no direct involvement or personal stake in the issue. Selecting a neutral and unbiased mediator helps make sure the resolution is fair and not swayed by personal opinions or interests. The mediator needs to be good at communicating to help everyone talk things through in a helpful way. Examples could be like a HR representative, a senior team member not connected to the project, or even an external consultant. Hope that helps!

Reply:
Thank you for the suggestion! I've added the points about neutrality and communication skills so that the reader does not think to bring in a mediator that favours their own side which was not my intent when writing.